test/basic.js

```diff
@@ -1,11 +1,10 @@
 var tape = require('tape')
-var memdown = require('memdown')
 var concat = require('concat-stream')
-var levelup = require('levelup')
+var level = require('level-mem')
```

Comment:
I prefer using `level-compose` because it's easier to take out `encoding-down` when needed. See https://github.com/vweevers/multileveldown/blob/wip-upgrade (and the discussion in https://github.com/Level/community/issues/71 that spurred it).